cmd/lvh/runner/qemu.go

```diff
@@ -39,6 +39,12 @@ func BuildQemuArgs(log *logrus.Logger, rcnf *RunConf) ([]string, error) {
 			fmt.Sprintf("telnet:localhost:%d,server,nowait", rcnf.SerialPort))
 	}
 
+	if rcnf.ConsoleLogFile != "" {
```

Comment:
Is it possible to pass two `-serial` args to qemu? If not, then we need to check whether `rcnf.SerialPort` is not set (can be done in a follow-up PR).

Reply:
Yep! I made sure to check this :). `-serial` can be used up to four times, check out https://man.archlinux.org/man/qemu.1.en#serial